### PR TITLE
create hyper-zoo-serving image

### PR DESCRIPTION
**Pull request by zzti-bsj**
* Dockerfile
* http-conf.yaml
* serving-conf.yaml
* start-cluster-serving.sh
* start-redis.sh
* start-http.sh
* download-cluster-serving-all-zip.sh

### CODE REVIEW

**Comment by glorysdj** [comment]
still need a check-status script to check redis/cluster serving/http statuses.

File: docker/hyperzoo-serving/start-client-serving.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (glorysdj) [inline docker/hyperzoo-serving/start-client-serving.sh:1]: better to rename to start-cluster-serving-all.sh

Comment (zzti-bsj) [reply docker/hyperzoo-serving/start-client-serving.sh:1]: ok

**Comment by glorysdj** [comment]
please also add a readme

**Pull request closed without merging**